cmd/skywire-cli/commands/visor/routes.go

```diff
@@ -22,6 +22,8 @@ func init() {
 		ruleCmd,
```

Comment:
I dont see a point for us to add get/set commands to the cli. The cli needs to run locally so the user might as well just change the config value himself. The http endpoint is needed to change this value remotely mostly.